src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -632,6 +646,24 @@ protected void Stop()
             RequestContext<object> requestContext)
         {
             // TODO: Handle this appropriately
+            //if (!this.noDebug)
+            //{
+            //    this.setBreakpointInProgress = true;
+
+            //    try
+            //    {
+            //        // Set exception breakpoints in DebugService
```

Comment:
Any reason we need this commented out code? If so, can you supply a reason in the comment?

Reply:
I'm hoping we will eventually get support for breaking on unhandled exceptions (maybe even first thrown).  When we do, I don't want to have the same mistake of not setting the setBreakpointInProgress flag.  That was the one of the causes of the crash.  I can add a comment along those lines.